tests/hikari/impl/test_rest.py

```diff
@@ -292,6 +292,199 @@ def test_invalidate_when_token_is_stored_token(self):
         assert strategy._expire_at == 0.0
         assert strategy._token is None
 
+#############################
+# OAuthCredentialsStrategy #
+#############################
+
+
+class TestOAuthCredentialsStrategy:
+    @pytest.fixture()
+    def mock_token(self):
+        return mock.Mock(
+            applications.PartialOAuth2Token,
```

Comment:
Spec setting is really inefficient, and unless we have a good reason to do so, shouldnt be doing it

Reply:
Could you explain what it means forspec setting?

Reply:
Removed the spec and tests still seem to work